contrib/devtools/github-merge.py

```diff
@@ -270,6 +266,11 @@ def main():
             print("Not signing off on merge, exiting.",file=stderr)
             exit(1)
 
+        second_sha512 = tree_sha512sum()
```

Comment:
I'd somewhat prefer to do this check

- after signing off on the merge
- but before putting the actual GPG signature

This avoids having a signed commit around (even though it's throwing it away) from something that isn't supposed to be.